cli/command/container/run_test.go

```diff
@@ -201,9 +201,7 @@ func TestRunAttachTermination(t *testing.T) {
 	case <-attachCh:
 	}
 
-	assert.NilError(t, syscall.Kill(syscall.Getpid(), syscall.SIGINT))
-	// end stream from "container" so that we'll detach
-	conn.Close()
```

Comment:
What was the reason for removing `conn.Close`?

Reply:
Is it still being flaky? From my testing closing the connection here didn't add anything.

Reply:
See: https://github.com/docker/cli/pull/5957

It's needed for the `cmdErrC` to actually receive the event. Removing it depended on the bug that was introduced.

I'm wondering if the flakiness was actually caused by the bug fixed by the mentioned PR 🤔